lib/ui/semantics.dart

```diff
@@ -366,6 +367,21 @@ class SemanticsFlag {
   /// used to implement accessibility scrolling on iOS.
   static const SemanticsFlag isHidden = const SemanticsFlag._(_kIsHiddenIndex);
 
+  /// Whether the semantics node is a live region.
+  /// 
+  /// A live region indicates that this semantics node will update its 
```

Comment:
This sounds misleading to me. Any semantics node may update its semantics label without being a liveRegion, no? liveRegion just tells a11y systems that a it may be relevant to inform the user about a changed label, no?